integrated-benchmarks/benchmarks.md

```diff
@@ -0,0 +1,10 @@
+<!-- README.md is generated from README.ecr, do not edit -->
```

Comment:
ditto - local machine initial data - let CI commit it in. 

Or check pipelines output @pichlermarc could you enable the runners for this PR?